test/unit/ui-test.js

```diff
@@ -389,30 +389,6 @@ describe('UI', function() {
         ui.destroy();
     });
 
-    it('triggers enter events with keyboard input', function() {
```

Comment:
Shouldn't this be changed to `'triggers click events with keyboard input'`? the test should register `on('click'` and then the rest should be mostly the same except that we expect `type: 'click'`.